outputs/opsgenie.go

```diff
@@ -0,0 +1,107 @@
+package outputs
+
+import (
+	"encoding/json"
+	"github.com/aquasecurity/postee/v2/formatting"
+	"github.com/aquasecurity/postee/v2/layout"
+	"log"
+
+	"github.com/opsgenie/opsgenie-go-sdk-v2/alert"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/client"
+)
+
+const defaultPriority = alert.P3
+
+type OpsGenieOutput struct {
+	Name       string
+	User       string
+	APIKey     string
+	Responders []string
+	VisibleTo  []string
+}
+
+func (ops *OpsGenieOutput) GetName() string {
+	return ops.Name
+}
+
+func (ops *OpsGenieOutput) Init() error {
+	log.Printf("Starting OpsGenie output %q....", ops.Name)
+	return nil
+}
+
+func getUserResponders(users []string) []alert.Responder {
+	if len(users) == 0 {
+		return nil
+	}
+	responders := []alert.Responder{}
+	for _, user := range users {
+		responder := alert.Responder{Type: alert.UserResponder, Username: user}
+		responders = append(responders, responder)
+	}
+	return responders
+}
+
+func getString(i interface{}) string {
+	if i == nil {
+		return ""
+	}
+	return i.(string)
+}
+
+func (ops *OpsGenieOutput) convertResultToOpsGenie(title string, content map[string]interface{}) *alert.CreateAlertRequest {
+	description := getString(content["description"])
+	alias := getString(content["alias"])
+	entity := getString(content["entity"])
+	priority := defaultPriority
+	if content["priority"] != nil {
+		priority = alert.Priority(getString(content["priority"]))
+	}
+	var tags []string
+	if content["tags"] != nil {
+		tags = content["tags"].([]string)
```

Comment:
This logic seems a little fragile to me. The user has freedom to input anything as we only expect an interface. 

Can we add some more validation of input to confirm it is of type `[]string`? A type check like the following might work:

```
	var tags []string
	if content["tags"] != nil {
		switch content["tags"].(type){
		case []string:
			tags = content["tags"].([]string)
		case string:
			tags = strings.Split(content["tags"].(string), ",")
		}
	}
```